src/test/test.ts

```diff
@@ -0,0 +1,124 @@
+
+import * as assert from "assert";
+import * as fs from "fs";
+import * as mocha from "mocha";
+import * as officeToolbox from "../util";
+import * as path from "path";
+import * as util from "util";
+const application = "excel";
+
+describe("Add manifest", function () {
+    describe(process.platform === "win32" ? "Add manifest to the registry" : "Add the manifest to the WEF folder", function () {
+        this.timeout(0);
+        let manifestFound = false;
+        let manifests: string[];
+        let testManifestPath = path.resolve(`${process.cwd()}/src/test/test-manifest.xml`);
+
+        it(process.platform === "win32" ? "Manifest should have been added to the registry" : "Manifest should have been added to the WEF folder and given a unique name", async function () {
+            await officeToolbox.addManifest(application, testManifestPath);
+
+            if (process.platform === "win32") {
+                manifests = await officeToolbox.getManifests(application);
+            } else {
+                testManifestPath = await officeToolbox.getSideloadManifestPath(testManifestPath, application);
+                manifests = await officeToolbox.getManifestsFromSideloadingDirectory(application);
+
+            }
+
+            manifests.forEach(element => {
+                if (element = testManifestPath) {
```

Comment:
one = only seems incorrect here

Reply:
should it validate the expected path?

Reply:
I don't following the first comment about 'one =only seems incorrect'

Reply:
one equal is assignment not test for equality.